### ISSUE TITLE
Color Contrast on code example gutters

### ISSUE THREAD

**Issue by 7mary4** [teachaccess/tutorial#103]
# Expected Behavior

The line numbers within a code example should have adequate color contrast
![line gutters with adequate contrast](https://user-images.githubusercontent.com/459953/151715397-a6c62da2-7056-42f3-bcc3-d1db89c31864.png)

## Suggested fix
Change the gutter text color to #707070
`.ͼ2 .cm-gutters {
  color: #707070;
}`

# Current Behavior

What is the current behavior?
The current line numbers use a light grey against an off-grey background
![code example with inadequate color contrast](https://user-images.githubusercontent.com/459953/151715441-5dd988b3-5acc-40fa-acf1-209a0ae199a8.png)

## Steps to reproduce

Please provide detailed steps for reproducing the issue.

1. visit 2.2 Headings
2. Compare the contrast of the code example numbers in the gutter. it should be 4.5:1
